ddtrace/contrib/redis/__init__.py

```diff
@@ -1,29 +1,44 @@
-"""Instrument redis to report Redis queries.
+"""
+Traces redis client queries.
+
+If you are not autoinstrumenting with ``ddtrace-run`` then install the redis
+instrumentation with::
+
+    from ddtrace import patch
+    patch(redis=True)
+
+
+Global Configuration
+~~~~~~~~~~~~~~~~~~~~
 
-``patch_all`` will automatically patch your Redis client to make it work.
-::
+.. py:data:: ddtrace.config.redis["service"]
```

Comment:
`.redis.service`?

Reply:
lol our convention is to use the dict access in the docs